[at_sp2_core] Unable to compile on i686

```
configure: error: Package requirements (xproto >= 7.0.13 x11 >= 1.6 xextproto >= 7.0.3 xext >= 1.0.99.1 inputproto >= 2.2.99.1) were not met:

No package 'xext' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables XI_CFLAGS
and XI_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
```

--- a/packages/at_spi2_core.rb
+++ b/packages/at_spi2_core.rb
@@ -12,6 +12,7 @@ class At_spi2_core < Package
   depends_on 'dbus'
   depends_on 'glib'
   depends_on 'gobject_introspection'
+  depends_on 'libxi'
   
   def self.build
     system "./autogen.sh"

